src/parallel/parallel.cpp

```diff
@@ -56,6 +58,9 @@ ParallelBucket::unlock()
 /// THREAD MANAGER
 ////////////////////////////////////////////////////////////////////////////////
 
+ThreadManager::ThreadManager(class ParallelRobinHoodHashTable *const hash_table)
+    : this->hash_table_(hash_table) {}
```

Comment:
```suggestion
    : hash_table_(hash_table) {}
```
You actually can't use `this` in the constructor's member initialiser list because it's specifically for member variables only.

Reply:
You learn something every day. I thought it looked strange... Thank you. 😄